galley/pkg/config/analysis/analyzers/schema/validation.go

```diff
@@ -37,7 +38,7 @@ func AllValidationAnalyzers() []analysis.Analyzer {
 	result := make([]analysis.Analyzer, 0)
 	for _, s := range schemas.Istio {
 		// Skip synthetic service entries
-		// TODO(https://github.com/istio/istio/issues/17949)
+		// See https://github.com/istio/istio/issues/17949
```

Comment:
https://github.com/istio/istio/issues/17949 was closed, and we don't necessarily want to remove this skip in the future, but leaving the reference in as explanatory context.